microservice/src/test/java/de/gsi/microservice/concepts/cmwlight/CmwLightExample.java

```diff
@@ -0,0 +1,243 @@
+package de.gsi.microservice.concepts.cmwlight;
+
+import static java.nio.file.Paths.get;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+import de.gsi.serializer.IoClassSerialiser;
+import de.gsi.serializer.spi.CmwLightSerialiser;
+import de.gsi.serializer.spi.FastByteBuffer;
+
+public class CmwLightExample {
```

Comment:
Codacy found an issue: [The utility class name 'CmwLightExample' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper|Constants)'](https://app.codacy.com/gh/GSI-CS-CO/chart-fx/pullRequest?prid=6417799)